UI changes

kyrers opened the pull request:
Hey all.

I saw the CTA to make the website prettier and decided to have a go at it.

I  enjoy simple websites, so I  just:
1. Changed a few things that were unpleasant to me, such as hovers, the header, and the dividers;
2. Fixed a bug with the input when on mobile.

Despite this having lots of changes, most of them are just formatting.
I also didn't delete anything as I didn't know the rationale behind leaving some unused stuff there.

Feel free to discard this or to suggest changes. Thanks!

kyrers committed:
UI changes

nanexcool commented:
hey, thank you for the PR, but you're changing way too many things at once, including the formatting of the code

kyrers committed:
Merge branch 'main' into ui-changes

kyrers committed:
Apply new changes

kyrers committed:
Remove formatting to avoid github detecting everything as a change

kyrers commented:
Apologies, I thought you'd prefer formatted code as it is easier to work with.

I merged the changes from main into this branch, changed the UI a little, fixed the input bug, and reverted the formatting as much as possible. It's much easier for you to see the changes now.
 
That said, the `styles.css` first line still shows lots of changes although there aren't that many, it's just that everything is on the same line, I don't think I can make it any less intrusive _(tip: if you compare the end of each line, you can quickly identify lines where there were changes and just check those_).

kyrers committed:
Remove forgotten comment

nanexcool commented:
Great! Thank you for the changes :) yeah, I realize minimized css will do that. Merging now!
